src/lib/expansion/expansion.md

```diff
@@ -0,0 +1,87 @@
+`<md-expansion-panel>` provides a expandable details-summary view.
```

Comment:
@willshowell, done. Take a look.

Also, the example plunk: https://plnkr.co/edit/5q98WJ?p=preview